docs/quickstart.md

```diff
@@ -88,14 +88,9 @@ you can follow the rest of the guide the same way as if not using docker.
 
 ## Preliminaries
 
-We have created a  <a href="https://github.com/fetchai/agents-template" target="_blank">template repo for AEA development</a> here which you can optionally fork and use immediately.
```

Comment:
Revert, this is optional.

Reply:
I directed the user to follow the `development-guide` page (which includes both manual and template-repo options). https://docs.fetch.ai/aea/development-setup/#approach-1

That's to avoid instruction duplication by reusing that guide. 

Do you mean I remove that line and put the instructions back?